app/code/Magento/InventorySourceSelectionApi/Model/GetSourceSelectionAlgorithmList.php

```diff
@@ -5,14 +5,16 @@
  */
 declare(strict_types=1);
 
-namespace Magento\InventorySourceSelection\Model;
+namespace Magento\InventorySourceSelectionApi\Model;
 
 use Magento\InventorySourceSelectionApi\Api\Data\SourceSelectionAlgorithmInterface;
 use Magento\InventorySourceSelectionApi\Api\Data\SourceSelectionAlgorithmInterfaceFactory;
 use Magento\InventorySourceSelectionApi\Api\GetSourceSelectionAlgorithmListInterface;
 
 /**
- * @inheritdoc
+ * {@inheritdoc}
+ *
+ * @api
  */
 class GetSourceSelectionAlgorithmList implements GetSourceSelectionAlgorithmListInterface
```

Comment:
Do we need to move implementation (like `GetSourceSelectionAlgorithmList `)  to `InventorySourceSelectionApi` namespace?
If we consider that GetSourceSelectionAlgorithmList is API, why do we need interface also?